examples/visualization/plot_study.ipynb

```diff
@@ -4,6 +4,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](http://colab.research.google.com/github/optuna/optuna/blob/master/examples/visualization/plot_study.ipynb)\n",
```

Comment:
I don't know the reason, but I encountered the following error when I clicked the button.

![image](https://user-images.githubusercontent.com/181413/69927601-dddfcd00-14fb-11ea-8a8a-df133fac361d.png)

![image](https://user-images.githubusercontent.com/181413/69927613-f05a0680-14fb-11ea-88d0-b1424c68a099.png)

Reply:
got the same issue as @sile and my chrome console says:

<img width="620" alt="Screen Shot 2019-12-02 at 12 55 38" src="https://user-images.githubusercontent.com/17039389/69929667-29e24000-1503-11ea-8ac9-e93027d70a46.png">

Reply:
Opening the link in a new tab worked.

Reply:
Seems like GitHub defaults to open the url in the preview iframe and fails. Can we default to open it in a new tab?

Reply:
I tested the html link below, but it didn't work.

```html
<a target="_blank" href="http://colab.research.google.com/github/optuna/optuna/blob/master/examples/visualization/plot_study.ipynb">
  <img src="https://colab.research.google.com/assets/colab-badge.svg">
</a>
```

https://github.com/harupy/optuna/blob/test-new-tab/examples/visualization/plot_study.ipynb

Chrome console says:

<img width="620" alt="Screen Shot 2019-12-02 at 13 59 02" src="https://user-images.githubusercontent.com/17039389/69931942-826a0b00-150c-11ea-92bc-d1b7876c6878.png">

Reply:
Oh, thanks for the investigation. I'm not an expert in this but it seems like this behavior is constrained by the attributes of the iframe which I don't think we can/should fiddle with https://stackoverflow.com/questions/21236616/sandboxed-iframe-with-target-blank-doesnt-open-new-tab-or-window/21236617. Should we leave a comment next to the button? I guess quite a few users will try to open it from GitHub.

Reply:
+1 on leaving a comment

Reply:
Thank you all for the investigation and comments.